src/main.rs

```diff
@@ -155,31 +172,26 @@ fn main() -> anyhow::Result<()> {
                         "Parsing",
                         format_args!("{crate_name} v{version} (current)"),
                     )?;
+
                     let rustdoc_path = rustdoc_cmd.dump(manifest_path, None, true)?;
+                    let current_crate = load_rustdoc(&rustdoc_path)?;
+
+                    // Can overwrite rustdoc of current crate.
+                    // For example, this happens when target-dir is specified in `.cargo/config.toml`.
+                    // That's the reason why we're immediately loading the rustdocs to memory.
```

Comment:
```suggestion
                    // The process of generating baseline rustdoc can overwrite 
                    // the already-generated rustdoc of the current crate.
                    // For example, this happens when target-dir is specified in `.cargo/config.toml`.
                    // That's the reason why we're immediately loading the rustdocs into memory.
                    // See: https://github.com/obi1kenobi/cargo-semver-checks/issues/269
```
More detail: what exactly causes the overwrite + link to the issue in question.